www_src/pages/project/project.jsx

```diff
@@ -37,7 +37,8 @@ var Project = React.createClass({
   },
 
   uri: function () {
-    return `/users/1/projects/${this.state.params.project}/pages`;
+    var user = this.state.user || {};
+    return `/users/${user.id}/projects/${this.state.params.project}/pages`;
```

Comment:
This breaks viewing projects from the discover list as we should be using the route params for both user id and project id. We should add a check to force the mode into "play" if the route param user id !== the session user id.